openedx_webhooks/tasks/github.py

```diff
@@ -413,17 +416,27 @@ def github_community_pr_comment(pull_request, jira_issue, people=None):
     people = {user.lower(): values for user, values in people.items()}
     pr_author = pull_request["user"]["login"].decode('utf-8').lower()
     created_at = parse_date(pull_request["created_at"]).replace(tzinfo=None)
+    missing_committers = []
```

Comment:
This should probably be a set, so that three commits by the same newb doesn't mention them three times in the comment.